app/modules/wallet/users/users.controller.ts

```diff
@@ -0,0 +1,53 @@
+import * as restify from 'restify';
```

Comment:
I'd change the filename to `authorized_users.controller.ts` and folder to `authorized_users`. Maybe these are more specific to the case?